test/setup-test-data.js

```diff
@@ -0,0 +1,35 @@
+const {account, payee, transactions} = require("./test-data")
+const {pensionAccount, currentAccount} = account
+
+const setupTestUser = async ({moneyhub}) => await moneyhub.registerUser({})
+const setupTestAccounts = async ({moneyhub, userId}) => {
+  return Promise.all([
+    moneyhub.createAccount({userId, account: currentAccount}),
+    moneyhub.createAccount({userId, account: pensionAccount})
+  ])
+}
+const setupPayee = async ({moneyhub, userId}) => await moneyhub.addPayee({
+  ...payee,
+  userId
+})
+const setupTransaction = async ({moneyhub, userId, accountId}) => {
+  return moneyhub.addTransactions({
+    userId,
+    transactions: transactions.map((trx) => ({...trx, accountId})),
```

Comment:
can we make sure that the created transactions have always the current date? that way we will not have to change the test data if the date ever gets too old

Reply:
Sure I can make the transactions have the current date. Not sure if this will break any tests but if it does I could try to use fake timers (not sure if this would work in the root hook). The tests that use the readOnlyUser are for getting counterparties, holdings, rental records and syncing a connection as these require a bit more complicated setup e.g. detecting regular transactions.

Reply:
let's try to use the current date in the transactions

maybe we could add in the tests docs the cases where the the readOnlyUser is used

Reply:
Cool I added that into the readme and the notion doc with the config.